mne/beamformer/tests/test_lcmv.py

```diff
@@ -288,6 +288,9 @@ def test_lcmv():
     assert 0.08 < tmax < 0.11, tmax
     assert 0.4 < np.max(max_stc) < 2., np.max(max_stc)
 
+    # Test if spatial filter contains src_type
+    assert ('src_type' in filters.keys())
```

Comment:
FYI you do not need parens here. (The only reason a lot of tests have these parens is that `'assert_true' -> 'assert '` was an easier find-and-replace in my editor than trying to remove the start and end parens.)

Reply:
Ah, I see! Thanks, good to know.

Reply:
`'src_type' in filters`

is enough

Reply:
you miss one @britta-wstnr